optaplanner-docs/src/modules/ROOT/pages/use-cases-and-examples/vaccination-scheduling/vaccination-scheduling.adoc

```diff
@@ -0,0 +1,266 @@
+[id="chap-optaplanner-vaccination_{context}"]
+= OptaPlanner Vaccination Scheduler
+
+:OPTAPLANNER-COMM:
+//Variable for the community version conditional statements.
+//:OPTAPLANNER-ENT:
+//Variable for the enterprise version conditional statements.
+
+ifdef::OPTAPLANNER-COMM[]
+
+:imagesdir: ../..
+
+:OPTAPLANNER: OptaPlanner
+:QUARKUS: Quarkus
+:OPTAPLANNER_JDK_VERSION: 11
+:OPTAPLANNER_MAVEN_VERSION: 3.6
+endif::OPTAPLANNER-COMM[]
+
+:context: optaplanner-vaccination-scheduler
+
+You can use the {OPTAPLANNER} vaccination appointment scheduler quick start to develop a vaccination schedule that is both efficient and fair. The vaccination appointment scheduler uses artificial intelligence (AI) to prioritize people and allocate time slots based on multiple constraints and priorities.
+
+.Prerequisites
```

Comment:
I'd move this prerequisite closer to where it's actually needed, which is the procedure **Downloading and running the OptaPlanner vaccination appointment scheduler**, so line 151.